src/phpcs/Production/Sniffs/Commenting/InjectSniff.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Prüft alle vorkommen von @Inject bzw. @Injectable auf korrekte Anwendung.
+ * Checks if the @Inject and @Injectable annotation is used the way we want it to.
```

Comment:
We want it to? Hätte es eher so formuliert: Checks the correct usage of @Inject and @Injectable annotations.